### PR TITLE
[REF] get rid of variable variable structure

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Readability improvement

Before
----------------------------------------
```php
$$date = 'blah'
```

After
----------------------------------------
```php
$params['start_date'] = 'blah'
```

Technical Details
----------------------------------------
Variable variables are hard to read

Comments
----------------------------------------

### CODE REVIEW

**Comment by colemanw** [comment]
retest this please

File: CRM/Member/BAO/Membership.php [hunk 1]
```diff
@@ -269,15 +268,11 @@ public static function create(&$params, &$ids = []) {
         $excludeIsAdmin = TRUE;
       }
 
-      $calcStatus = CRM_Member_BAO_MembershipStatus::getMembershipStatusByDate($start_date, $end_date, $join_date,
+      $calcStatus = CRM_Member_BAO_MembershipStatus::getMembershipStatusByDate($params['start_date'], $params['$end_date'], $params['join_date'],
```
Comment (seamuslee001) [inline CRM/Member/BAO/Membership.php:271]: @eileenmcnaughton there is a stray $ in the end_date

File: tests/phpunit/api/v3/ContactTest.php [hunk 2]
```diff
@@ -1721,17 +1726,15 @@ public function testMergeOrganizations() {
...
+      'join_date' => '01/01/2010',
+      'end_date' => '12/31/2015',
     ];
-    $ownermembershipid = $this->contactMembershipCreate($membershipParams);
+    $ownerMembershipID = $this->contactMembershipCreate($membershipParams);
 
-    $contactmembership = $this->callAPISuccess("membership", "getsingle", [
-      "contact_id" => $contact["id"],
-    ]);
+    $contactmembership = $this->callAPISuccessGetSingle('membership', ['contact_id' => $contact["id"],]);
```
Comment (seamuslee001) [inline tests/phpunit/api/v3/ContactTest.php:1735]: minor thing but why the trailing `,? @eileenmcnaughton `

Comment (eileenmcnaughton) [reply tests/phpunit/api/v3/ContactTest.php]: Jenkins didn't think it was minor

**Pull request merged**